src/utils.js

```diff
@@ -126,14 +127,23 @@ export function getVariable(context, name) {
  *
  */
 export function gettext(str) {
-  return str;
+  const locale = process.env.LANG || process.env.LANGUAGE || process.env.LC_ALL || process.env.LC_MESSAGES || 'en-US';
+  let i18ndata = {};
+  try {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    i18ndata = require(`./locale/${locale}/messages.js`);
+  } catch (err) {
+    log.info('Initialize locales using extract-locales command');
+  }
+  const jed = new Jed(i18ndata);
+  return jed.gettext(str);
```

Comment:
I actually wonder why you moved this into the gettext function again. We have the language set on a process basis which means that it won't change during the whole life-time of the addons-linter process.

What's happening here now is that *every* `gettext` call re-initializes `Jed` which might actually lead to quite some slow-down. It's not super massive but moving that into our global state will probably speed things up.

Reply:
Yes that is right @EnTeQuAk 

But If I make the jed variable global to prevent re initialization, a few tests fail in which the global jed variable comes in as undefined inside the gettext function.